mempool/templates/frontend-deployment.yaml

```diff
@@ -54,10 +54,10 @@ spec:
             {{- toYaml .Values.frontend.securityContext | nindent 12 }}
           image: "{{ .Values.frontend.image.repository }}:{{ .Values.frontend.image.tag }}"
           imagePullPolicy: {{ .Values.frontend.image.pullPolicy }}
-          command:
-            - ./wait-for {{ template "mempool.name" . }}-mariadb:3306
-            - --timeout=720
-            - -- nginx -g 'daemon off;'
+          # command:
+          #   - ./wait-for {{ template "mempool.name" . }}-mariadb:3306
+          #   - --timeout=720
+          #   - -- nginx -g 'daemon off;'
```

Comment:
What does this do and do we need it?

Reply:
Doesn't seem like this is needed. Removing. Backend will wait for mariadb to be ready and nginx command is already the CMD in the docker image.